app/assets/javascripts/room.js

```diff
@@ -127,6 +127,9 @@ $(document).on('turbolinks:load', function(){
 });
 
 function showCreateRoom(target) {
+  $('#createRoomModal').on('shown.bs.modal', function () {
```

Comment:
Good implementation, but not the right place to add it. Might want to move it to below the `#create-room-block` click handler